python/gen_enums.py

```diff
@@ -0,0 +1,37 @@
+import sys
+from pathlib import Path
+
+from google.protobuf.descriptor import EnumDescriptor
+
+
+def main(input_module_file: Path, output_file: Path):
+    src = """# This file is auto-generated from gen_enums.py
+from enum import Enum
+"""
+
+    input_module_src = input_module_file.read_text('utf-8')
+    input_module = dict()
+    exec(input_module_src, input_module)
```

Comment:
This cannot be done because of the chicken and an egg problem - importlib loads modules importing the enum package. That can work only if the file is already present and has valid entries.